include/oneapi/tbb/task_group.h

```diff
@@ -234,7 +233,7 @@ class task_group_context : no_copy {
     };
 
     //! Thread data instance that registered this context in its list.
-    std::atomic<r1::thread_data*> my_owner;
+    r1::context_list_control* my_context_list_control;
```

Comment:
You can just left `r1::context_list* my_context_list;`, because it's list + mutex -> "concurrent list"